tests/difference/core/model/src/constants.ts

```diff
@@ -42,7 +42,11 @@ const MODEL_INIT_STATE: ModelInitState = {
   h: { provider: 0, consumer: 0 },
   t: { provider: 0, consumer: 0 },
   ccvC: {
-    hToVscID: { 0: 0, 1: 0 },
+    // The initial provider height is 0 but there must have been
```

Comment:
Could you give more context in this comment?

Maybe explain why a "phantom" vscId is needed?

Reply:
Good point! Please see change.